api/src/resources/validation.rs

```diff
@@ -19,9 +19,58 @@ pub struct LabelValidationResponse {
     pub label_validation: LabelValidation,
 }
 
+#[derive(Debug, Clone, Deserialize, Serialize, PartialEq)]
+pub struct ModelScore(pub f32);
+
+impl std::fmt::Display for ModelScore {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(f, "{}", self.0)
+    }
+}
```

Comment:
fyi this looks like the kind of thing you could use the `derive_more` crate to do: https://jeltef.github.io/derive_more/derive_more/#example-code